Yubico.YubiKey/src/Yubico/YubiKey/IYubiKeyDeviceInfo.cs

```diff
@@ -126,5 +143,15 @@ public interface IYubiKeyDeviceInfo
         /// Indicates if this device has temporarily disabled NFC.
         /// </summary>
         public bool IsNfcRestricted { get; }
+
+        /// <summary>
+        /// The chip number, if available, otherwise empty string.
+        /// </summary>
+        string PartNumber { get; }
```

Comment:
We went with the empty string initially as the firmware on newer keys always returns the tag, but empty for non-FIPS. After a quick discussion we decided to go the other way, and map the empty TLV to null. This to avoid a situation where < 5.7 keys present NULL, but <= 5.7 present `""`.

Reply:
I think that matches my expectations. 👍 

Just extra background: I typically am very hesitant to rely on nullable things in C# - people tend to over-use and misuse it. It's meant to represent the absence of something, which in this case matches perfectly the intent.

Reply:
I appreciate the added extra background. I tend to avoid using nullabe fields if I can.